iothub_client/tests/common_longhaul/iothub_client_common_longhaul.c

```diff
@@ -51,7 +52,8 @@ static const char* IOTHUB_LONGHAUL_LOOP_DURATION_SECS = "IOTHUB_LONGHAUL_LOOP_DU
 #define DEVICE_METHOD_SUB_WAIT_TIME_MS          (5 * 1000)
 #define NETWORK_RETRY_ATTEMPTS                  20
 #define SERVICE_NETWORK_RETRY_ATTEMPTS          4
-#define NETWORK_RETRY_DELAY_MSEC                5 * 1000
+#define NETWORK_RETRY_DELAY_MSEC                15 * 1000
```

Comment:
Nit - values like this in a #define should have a parathethesis just in case there's order of operation issues, so `(15*1000)`.